applications/wg-easy/Taskfile.yaml

```diff
@@ -177,6 +206,17 @@ tasks:
         fi
       - echo "All matching clusters deleted and kubeconfig cleaned up!"
 
+  update-version:
```

Comment:
I'd like some conversation / explanation of what the intorduction of .env is how/why you see updating it like this etc. This might be a good pattern it's just not immediately obvious to me so I could use some explanation.

I tend to treat 'versions' as git tags. That's to say that the version is tracked with the source control and not files within it. In particular the concern I have with having the version _in_ the repo is it seems to regularly end up in a situation where you need to release a new version like "RC1" moving to "Stable" where you don't want code changes, but now you _have_ to change code todo that.

Reply:
This one, I am trying to aline with our makefile create-release cmd. You are right, let me know rethink it.

Reply:
the problem in a repo like platform-examples is that the git tag is for the entire repo, and here we want to version each of the example applications.  I'd love to know a way we can say that wg-easy is version 1.2.3 but some other app is 0.4.1 and so on.